packages/web/src/components/SignInForm/styles.scss

```diff
@@ -4,9 +4,12 @@
 
 form {
   width: 70%;
-  margin-bottom: 76px;
 
   .register {
     margin-top: 16px;
   }
 }
+
+form.signinForm {
+  margin-bottom: 20px;
+}
```

Comment:
Why did you remove my 76px margin? And why is it now 20px?
The idea was that it compensated for a notification placeholder above, so it doesn't jump and  it vertically centered.
Is it for the additional facebook button?

Reply:
Yep. I have tried wrong email for this page and after notification have not noticed any "jumps". Mb new container (i.e. `div className="conteiner ..`) solves the problem